doc/developers/advanced_installation.rst

```diff
@@ -226,8 +226,8 @@ macOS compilers from conda-forge
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you use the conda package manager (version >= 4.7), you can install the
-``compilers`` meta-package from the conda-forge channel, which provides
-OpenMP-enabled C/C++ compilers based on the llvm toolchain.
+``compilers`` meta-package from the conda-forge channel (use version >= 1.0.4)
+which provides OpenMP-enabled C/C++ compilers based on the llvm toolchain.
```

Comment:
Please put the version requirement in the conda create command instead: ``conda create ... conda-forge::compilers>=1.0.4`` so that people can copy paste safely

Reply:
Also can you test the new command locally and confirm it works ?

Reply:
sure thing, we have to use double quotes `""` to specify the version in CLI

Reply:
it works fine locally 👌